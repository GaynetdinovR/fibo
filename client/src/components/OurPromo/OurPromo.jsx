import styles from "../../styles/components/OurPromo.module.sass";
import H2 from "../../ui/H2.jsx";
import OvalButton from "../../ui/OvalButton.jsx";

const OurPromo = () => {
	const promos = [
		{ src: "./content/promo_1.jpg" },
		{ src: "./content/promo_2.jpg" },
		{ src: "./content/promo_3.jpg" },
		{ src: "./content/promo_1.jpg" },
		{ src: "./content/promo_4.jpg" }
	];

	return (
		<section className={styles.our_promo}>
			<H2 className={styles.our_promo__title}>
				<span>Наши </span>
				<span>акции</span>
			</H2>
			<div className={styles.our_promo__content}>
				{promos.map((promo, i) => (
					<div key={i} className={styles.our_promo__promo}>
						<img src={promo.src} alt="promo" />
					</div>
				))}
			</div>
			<OvalButton className={styles.our_promo__all_promo_btn}>
				Все акции
			</OvalButton>
		</section>
	);
};

export default OurPromo;