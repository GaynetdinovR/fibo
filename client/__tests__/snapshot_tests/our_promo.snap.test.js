import renderer from "react-test-renderer";
import { it, expect } from "@jest/globals";
import OurPromo from "../../src/components/OurPromo/OurPromo.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<OurPromo/>).toJSON();
	expect(tree).toMatchSnapshot();
});
