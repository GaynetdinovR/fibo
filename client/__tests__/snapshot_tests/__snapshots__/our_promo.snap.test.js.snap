// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="our_promo"
>
  <h2
    className="our_promo__title h2"
  >
    <span>
      Наши 
    </span>
    <span>
      акции
    </span>
  </h2>
  <div
    className="our_promo__content"
  >
    <div
      className="our_promo__promo"
    >
      <img
        alt="promo"
        src="./content/promo_1.jpg"
      />
    </div>
    <div
      className="our_promo__promo"
    >
      <img
        alt="promo"
        src="./content/promo_2.jpg"
      />
    </div>
    <div
      className="our_promo__promo"
    >
      <img
        alt="promo"
        src="./content/promo_3.jpg"
      />
    </div>
    <div
      className="our_promo__promo"
    >
      <img
        alt="promo"
        src="./content/promo_1.jpg"
      />
    </div>
    <div
      className="our_promo__promo"
    >
      <img
        alt="promo"
        src="./content/promo_4.jpg"
      />
    </div>
  </div>
  <button
    className="our_promo__all_promo_btn oval_btn"
  >
    Все акции
  </button>
</section>
`;
